android/app/build.gradle

```diff
@@ -22,6 +22,12 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
 android {
     namespace "com.example.app_buy_sell"
```

Comment:
As far as I know, Google doesn't allow us to use "example" in bundle id. Could you change it to com.app_buy_sell.release?